connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -729,6 +736,18 @@ message WithColumns {
   repeated Expression.Alias aliases = 2;
 }
 
+message WithWatermark {
+
+  // (Required) The input relation
+  Relation input = 1;
+
+  // (Required)
+  string event_time = 2;
```

Comment:
string or long?

Reply:
Updated the the comment. API documentation is the full documentation.